curriculum/challenges/english/25-front-end-development/lab-first-element-finder/a6e40f1041b06c996f7b2406.md

```diff
@@ -0,0 +1,124 @@
+---
+id: a6e40f1041b06c996f7b2406
+title: Build a First Element Finder
+challengeType: 26
+dashedName: build-a-first-element-finder
+---
+
+# --description--
+
+In this lab, you will create a function that looks through an array and returns the first element that passes a test function (also known as a 'truth' test). 
+
+The function whould iterate through the array and test each element using the provided test function. At the end, it would return the first element where the test function returns `true`.
```

Comment:
```suggestion
The function would iterate through the array and test each element using the provided test function. At the end, it would return the first element where the test function returns `true`.
```